scripts/dev-setup.sh

```diff
@@ -1,5 +1,7 @@
 # Install empty Drupal site with user 1 as admin/admin
-ddev drush si -y --account-pass=admin --site-name='Labdoo3.0' standard
+ddev drush si -y --account-pass=admin --site-name='Labdoo3.0' minimal
+# Force the new site UUID to be the same as the configuration
+ddev drush config:set 'system.site' uuid 'e5f3dd3c-79fa-4957-84db-90fc98effac5' -y
```

Comment:
This appears to be setting the site unique ID to the same value in system.site.yml:

https://github.com/greggmarshall/Labdoo-3.0/blob/main/config/default/system.site.yml

I am guessing this is needed as a safety check to avoid importing from a site that is different, as explained here:

https://www.hashbangcode.com/article/drupal-8-configuration-originates-different-site